tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/sparse-config/test-suite-a/testfunctionreturningintersectiontype-function.html

```diff
@@ -1,47 +1,24 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
   </head>
   <body>
     <section>
-      <h2>
-        testFunctionReturningIntersectionType
-      </h2>
+      <h2>testFunctionReturningIntersectionType</h2>
       <section>
-        <p>
-          Test function that returns an inline type
-        </p>
+        <p>Test function that returns an inline type</p>
       </section>
       <section>
-        <p>
-          <span><b>WARNING: This API is deprecated and will be removed in a future release.</b></span>
-          <br>
-          <span>
-          <p>
-            <i>This is a test deprecation notice. Here is a</i> <a href='docs/test-suite-a/testfunctionreturninguniontype-function'><i>link</i></a> <i>to something else!</i>
-          </p>
-          </span>
-        </p>
+        <p><span><b>WARNING: This API is deprecated and will be removed in a future release.</b></span><br><span><i><p>This is a test deprecation notice. Here is a <a href="docs/test-suite-a/testfunctionreturninguniontype-function">link</a> to something else!</p></i></span></p>
```

Comment:
Here the second paragraph got italicized as a whole, whereas only the non-`<a>` text was before. I think the new version is fine... but in case you want to look into it.

Not a fan of multiple `<p>`s in a single line, but another thing I wouldn't block on; I'd just be happier if we can configure the behavior to not do it. And I see in lines 20-21 below that it can render them in separate lines, so this feels weird.

Reply:
> Here the second paragraph got italicized as a whole, whereas only the non-<a> text was before. I think the new version is fine... but in case you want to look into it.

This appears to have been a bug in our previous rendering code. The link _should_ have respected the formatting of the containing span.

> Not a fan of multiple <p>s in a single line, but another thing I wouldn't block on; I'd just be happier if we can configure the behavior to not do it. And I see in lines 20-21 below that it can render them in separate lines, so this feels weird.

I agree. But I'm viewing this renderer as a default implementation that users can take or leave. So I'm not sure I personally care too much what the output looks like. Users can leverage the `hast` output and call / implement their own renderers as they see fit.

For fluidframework.com, we only leverage this functionality for tables, and I was intending to likely not opt-in to formatted rendering long term anyways for performance reasons.